src/App.tsx

```diff
@@ -15,71 +17,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
You have the `Jam` value in the goods array already 
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```